### ISSUE TITLE
Store size of package in the index

### ISSUE THREAD

**Issue by aslozada** [fortran-lang/fpm-registry#35]
Should the ```fpm.toml``` file contain information about the size of the package?  

The subroutines ```new_package``` and ```check```, in  the module ```fpm_manifest_package```, contain 18 keywords with information about a package, ```name```, ```author```, ```version```, etc. However, no keyword provides information about the size of the package.  

An idea,
1. Build the package
2. Update the file ```fpm.toml``` with information about the size of the package

The command ```execute_command_line``` can get this information from shell.

**Comment by certik** [comment]
Indeed, the fpm-registry should automatically extract this information and make it available.
